DER/FxDerivatives/FxContracts.rdf

```diff
@@ -75,132 +75,134 @@
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;Currency"/>
-		<skos:definition xml:lang="en">The buyer&apos;s currency as specified in the contract.</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-ctr;hasExchangedCurrencyOne">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasCurrency"/>
-		<rdfs:label xml:lang="en">has exchanged currency one</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForward"/>
+		<rdfs:label xml:lang="en">has buying currency</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;CurrencyDerivative"/>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;Currency"/>
-		<skos:definition xml:lang="en">FpML has a term exchangedCurrency1 but this refers to the currency flow not the currency as in USD, CDN etc. FpML definition for exchangedCurrency1: &apos;This is the first of the two currency flows that define a single leg of a standard foreign exchange transaction.&apos;</skos:definition>
+		<skos:definition xml:lang="en">indicates the currency in which the option is bought</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-ctr;hasExchangedCurrencyTwo">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasCurrency"/>
-		<rdfs:label xml:lang="en">has exchanged currency two</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForward"/>
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;Currency"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-ctr;hasForeignExchangeForwardRate">
+	<owl:ObjectProperty rdf:about="&fibo-der-fx-ctr;hasForwardExchangeRate">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-qt-qtu;hasQuantityValue"/>
-		<rdfs:label xml:lang="en">has foreign exchange forward rate</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForwardOutright"/>
+		<rdfs:label xml:lang="en">has forward exchange rate</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;CurrencyForward"/>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;ExchangeRate"/>
-		<skos:definition xml:lang="en">The rate at which the seller currency is purchased in the buyer currency.</skos:definition>
+		<skos:definition xml:lang="en">rate of exchange between two currencies as specified in a forward contract</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-fx-ctr;hasForeignExchangeForwardRateValue">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasRateValue"/>
-		<rdfs:label xml:lang="en">has foreign exchange forward rate value</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForward"/>
-		<rdfs:range rdf:resource="&xsd;decimal"/>
-		<skos:definition xml:lang="en">&apos;The rate of exchange between the two currencies.&apos;</skos:definition>
-	</owl:DatatypeProperty>
+	<owl:ObjectProperty rdf:about="&fibo-der-fx-ctr;hasSellingCurrency">
```

Comment:
I think the option itself is bought/sold in one currency. Which is distinct from the currencies of the exchange. e.g. I could buy an option for 100 USD to exchange 5000 GBP for 4500 Euros on June 1.

Reply:
I've revised the definition and added a note - the restrictions are on all currency derivatives, including spots, so the definitions were not right in any case ... something I had not changed, leftover from MB's original work